### PR TITLE
feat(device): add `supports_volume` field

### PR DESCRIPTION
**Pull request by ale-vncs**
Adding the "supports_volume" field in Device.java

https://developer.spotify.com/documentation/web-api/reference/get-a-users-available-devices

### CODE REVIEW

File: src/main/java/se/michaelthelin/spotify/model_objects/miscellaneous/Device.java [hunk 1]
```diff
@@ -19,6 +19,7 @@ public class Device extends AbstractModelObject {
   private final String name;
   private final String type;
   private final Integer volume_percent;
+  private final Boolean supports_volume;
```
Comment (dargmuesli) [inline src/main/java/se/michaelthelin/spotify/model_objects/miscellaneous/Device.java:22]: Would you mind keeping the alphabetical sorting of properties here and in other places?

**Review by dargmuesli — COMMENTED** [review]
Thank you for that addition!

File: src/main/java/se/michaelthelin/spotify/model_objects/miscellaneous/Device.java (hunk 1 above)
Comment (ale-vncs) [reply src/main/java/se/michaelthelin/spotify/model_objects/miscellaneous/Device.java:22]: No, I can do :).

But what be "other places", just `Device.java`?

Comment (dargmuesli) [reply src/main/java/se/michaelthelin/spotify/model_objects/miscellaneous/Device.java:22]: Yes

Comment (ale-vncs) [reply src/main/java/se/michaelthelin/spotify/model_objects/miscellaneous/Device.java]: Done

**Review by dargmuesli — APPROVED** [review]
Perfect!

**Pull request merged**